### PR TITLE
adding stimulus links and info to readme

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | none
| License       | MIT

Hi!

I mostly wanted to add links to other Stimulus libraries. I updated the top because, while it was great for release, it doesn't explain what UX is. And, I'm not even sure it should - it should probably be moved more into the docs. We can chat about this part :).

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,23 @@
...
-in your pages, and to make your front-end code interact with the rest of your
-infrastructure.
-
-This problem is not new: it looks a whole lot like the state of Symfony in PHP
-before Symfony Flex. We need a Symfony Flex equivalent for JavaScript: a tool to
-build amazing User Experiences as quickly as we can now setup
-[an HTTP client](https://symfony.com/doc/current/http_client.html),
-[a Mailer](https://symfony.com/doc/current/mailer.html) or
-[an administration panel](https://symfony.com/doc/current/bundles/EasyAdminBundle/index.html).
+Symfony UX is an initiative and set of libraries for seamlessly
```
Comment (tgalopin) [inline README.md:9]: `to seamlessly integrate`? I've no idea why but I feel that this wording is weird. Probably only me :D

File: README.md [hunk 2]
```diff
@@ -6,27 +6,23 @@
...
+integrating JavaScript tools into your application. For example,
+want to render a chart with [Chart.js](https://www.chartjs.org/)? Use
```
Comment (tgalopin) [inline README.md:11]: I'm not sure about this sentence, it feels a bit too familiar for the official README?

Comment (weaverryan) [reply README.md:11]: I hear what you're saying... but the alternative is to make it more boring sounding... which I don't love. This quickly delivers the benefit of UX. But, I'm open to suggestions.

**Comment by tgalopin** [comment]
Thanks Ryan.

**Pull request merged**